README.md

```diff
@@ -116,90 +116,90 @@ nlsolve(f!, initial_x, autodiff = true)
 ### Jacobian available
 
 If, in addition to `f!(x::AbstractVector, fx::AbstractVector)`, you have a
-function `g!(x::AbstractVector, gx::AbstractMatrix)` for computing the Jacobian
+function `j!(x::AbstractVector, gx::AbstractMatrix)` for computing the Jacobian
```

Comment:
+1 I like this semantic change.

Reply:
I see I missed `gx` in the positional argument in the README, but yeah, `g` was always "confusing" tome.